inscriptions/ISic000503.xml

```diff
@@ -196,14 +201,18 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la" resp="#JH">
+                <ab>
+                    <lb n="1"/><persName><name><expan><abbr>L</abbr><ex>ucius</ex></expan> · Pinarius · Rufus</name></persName> · ob · honorem
+                    <lb n="2"/>aedilitatis · imaginem · gen<supplied reason="lost">ii</supplied>
+                    <lb n="3"/>municipi · Lilybitanorum · ex · <choice><orig>argr</orig><orig>·</orig><orig>vps</orig><orig>a</orig><reg>arg. PVPSP</reg></choice>
```

Comment:
Last line is tricky. However, we're not using 'choice' like this, only for orthographic variation on individual words. More complex issues of reading simply get reported in the apparatus as comments to the individual lines, in the style of a 'traditional' apparatus, although you don't need to worry about this at this point.
This is a nice case of a now lost text, reported only by Gualtherus and subsequent editors trying to make sense of it. Gualtherus' reading was this:
![image](https://user-images.githubusercontent.com/25080386/110446111-b8eee800-80b6-11eb-85d8-61b426733182.png)

As you can see, firstly, you actually want to report Lilybitanor(um) as an expanded abbreviation (oddly missed by Mommsen). and then secondly, we have to make sense of ARGR. VP S P (whatever is meant by the line through the final P).
The best effort of modern editors has been:
ex arg(ento) p(ondo) V p(ecunia) s(ua) p(osuit)
To render this correctly, I think one must simply expand the abbreviation as it's presented and then add an apparatus note as suggested below, since otherwise it becomes almost impossibly complex and not worth the effort.